components/DaysOfTheWeek/DaysOfTheWeekContainer.jsx

```diff
@@ -0,0 +1,122 @@
+import { useState } from "react";
+import { Text, Box, SimpleGrid, GridItem, Flex } from "@chakra-ui/react";
+import DaysOfWeekButton from "./DayOfTheWeekButton";
+import LinkButton from "../LinkButton/LinkButton";
+
+export default function DaysOfTheWeekContainer ({ setNumberOfDays, onSaveEvent }) {
+
+  // initial state for days of the week has info if it's selected or not (instead of having 2 separate states)
+  const [ daysOfTheWeek, setDaysOfTheWeek ] = useState([
+    {
+      id: 0,
+      day: "Monday",
+      isSelected: false
+    },
+    {
+      id: 1,
+      day: "Tuesday",
+      isSelected: false
+    },
+    {
+      id: 2,
+      day: "Wednesday",
+      isSelected: false
+    },
+    {
+      id: 3,
+      day: "Thursday",
+      isSelected: false
+    },
+    {
+      id: 4,
+      day: "Friday",
+      isSelected: false
+    },
+    {
+      id: 5,
+      day: "Saturday",
+      isSelected: false
+    },
+    {
+      id: 6,
+      day: "Sunday",
+      isSelected: false
+    }
+  ]);
+
+  // state to activate and disable save button (link button)
+  const [ saveButtonActive, setSaveButtonActive ] = useState(true);
+
+  // on click, the buttom will change colour and the state will be updated to include selected buttons
+  const handleClick = (e) => {
+    e.preventDefault();
+
+    let updatedData = [...daysOfTheWeek];
+
+    // based on the day name from innerHTML of each button, update the state if selected or unselected
+    updatedData.map(item => {
+      if (item.day === e.target.innerText) {
+        item.isSelected = item.isSelected ? false : true;
+      }
+    })
+    setDaysOfTheWeek(updatedData);
+
+    // activate Save button (link button) when something is selected
+    const areDaysSelected = updatedData.find(item => item.isSelected) ? false : true;
+    setSaveButtonActive(areDaysSelected);
+
+    // get answer to how many days a week user works
+    const workingDaysNumber = updatedData.filter(item => item.isSelected).length;
+    console.log(workingDaysNumber)
+    setNumberOfDays(workingDaysNumber)
+  }
+
+ // NOTE! I use minW attribute for the Box for now because at the moment the layout component has limitation for width. 
+ // Please change once Layout is changed
+
+  return (
+    <Box 
+      justify="center" 
+      borderWidth="2px" 
+      borderRadius="lg" 
+      minW="750px" 
```

Comment:
For context to other reviewers:
if `minW` isn't set, there won't be enough room to place 4 'days of the week' buttons on the same row, resulting in overlapping buttons: 
![image](https://user-images.githubusercontent.com/4408259/157127132-c16802d1-191d-446e-9d9d-be63c0ed5ee7.png)

Reply:
* Note is saved as a message in Trello under the related ticket => [<a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;Nuhvxjes&#x2F;299-future-issues-to-address">Future issues to address</a></blockquote>](https://trello.com/c/Nuhvxjes)
* Separate ticket is created => <a href="https:&#x2F;&#x2F;trello.com&#x2F;c&#x2F;635h1sLO&#x2F;298-create-a-component-and-associated-page-to-select-a-day-of-the-week-as-per-design">Create a component and associated page to select a day of the week as per design.</a></blockquote>